packages/sdk/src/core/utils/process.ts

```diff
@@ -0,0 +1,15 @@
+export function getProcessEnv(key: string, defaultValue = "") {
+  if (typeof process !== undefined) {
```

Comment:
```suggestion
  if (typeof process !== "undefined") {
```

gotta compare to string value (typeof returns a string)